src/components/Table/Table.jsx

```diff
@@ -89,9 +89,9 @@ const TableCreator = ({
               }
 
               let fieldEl = null;
-              const bars = relativeBars || percentBars;
+              const bars = relativeBars || percentBars || percentBarsWithValue;
               if (bars) {
-                const altValue = typeof bars === 'function' ? bars(row) : null;
+                const altValue = typeof bars === 'function' && percentBarsWithValue ? bars(row) : null;
```

Comment:
I don't know if this is needed anymore. I'm assuming the withValue should be the only one that's a func